bin/run-clang-format.sh

```diff
@@ -14,4 +14,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# Check if clang-format is installed
+if ! command -v clang-format >/dev/null 2>&1; then
+    echo "Error: clang-format is not installed"
+    echo "Please install it using: brew install clang-format"
+    exit 1
+fi
```

Comment:
I can remove this - but it helped me figure out clang-format wasn't installed through brew on my mac.